stories/util.js

```diff
@@ -112,7 +116,10 @@ export const WrapWithHits = ({
   };
 
   return (
-    <InstantSearch appId={appId} apiKey={apiKey} indexName={indexName}>
+    <InstantSearch
+      searchClient={algoliasearch(appId, apiKey)}
```

Comment:
This can be stored in the React component otherwise it creates a new search client at every render.